.buildkite/pipelines/quality-gates/pipeline.tests-qa.yaml

```diff
@@ -3,37 +3,54 @@
 # this pipeline build will prevent further progression to the subsequent stage.
 
 steps:
-  - label: ":pipeline::kibana::seedling: Trigger Kibana Tests for ${ENVIRONMENT}"
-    command: echo "replace me with Kibana specific tests"
-    agents:
-      image: "docker.elastic.co/ci-agent-images/basic-buildkite-agent:1688566364"
+  - label: ":pipeline::kibana::seedling: Trigger Kibana Serverless Tests for ${ENVIRONMENT}"
+    trigger: appex-qa-kibana-serverless-ftr-tests # https://buildkite.com/elastic/appex-qa-kibana-serverless-ftr-tests
+    build:
+      env:
+        ENVIRONMENT: ${ENVIRONMENT}
+      message: "${BUILDKITE_MESSAGE} (triggered by pipeline.tests-qa.yaml)"
 
-  - label: ":pipeline::fleet::seedling: Trigger Fleet Kibana Tests for ${ENVIRONMENT}"
-    command: echo "replace me with Fleet specific Kibana tests"
-    agents:
-      image: "docker.elastic.co/ci-agent-images/basic-buildkite-agent:1688566364"
+  - label: ":pipeline::weight_lifter::seedling: Trigger RAC Alert Load tests for ${ENVIRONMENT}"
+    trigger: appex-qa-rac-alert-load # https://buildkite.com/elastic/appex-qa-rac-alert-load
+    build:
+      env:
+        ENVIRONMENT: ${ENVIRONMENT}
+        SERVERLESS_ONLY: true
+      message: "${BUILDKITE_MESSAGE} (triggered by pipeline.tests-qa.yaml)"
```

Comment:
@delanni I chatted with Robert over Slack and we decided for now to simply remove this step. As the load tests are currently they are not adding any value to the QA team and even as soft fail it would just be too noisy

Reply:
Yes, we discussed this as a group and agreed that we want to move in iterations here. Having a smaller set of tests in the first iteration allows us to stabilize them faster. So we decided to leave away the load tests and cypress tests for this initial round.

Reply:
All right! noted, I'll remove these tests.